src/lib.rs

```diff
@@ -754,7 +754,7 @@ impl IngestProcessor {
         let dlq = dead_letter_queue_from_options(&opts).await?;
         let transformer = Transformer::from_transforms(&opts.transforms)?;
         let table = delta_helpers::load_table(table_uri, HashMap::new()).await?;
-        let coercion_tree = coercions::create_coercion_tree(&table.get_metadata()?.schema);
+        let coercion_tree = coercions::create_coercion_tree(table.schema().unwrap());
```

Comment:
Why switch from `?` to `unwrap`? While this _shouldn't_ really be a candidate for panicking, it is a possibility. Is that what we want?